vm-setup/roles/v1aX_integration_test/tasks/move.yml

```diff
@@ -43,13 +43,15 @@
 
   # Install BMO
   - name: Install Baremetal Operator
-    shell: "kustomize build /home/$USER/go/src/github.com/metal3-io/baremetal-operator/deploy/default/ | kubectl --kubeconfig=/tmp/kubeconfig-{{ CLUSTER_NAME }}.yaml apply -n {{ NAMESPACE }} -f -"
+    shell: "kustomize build /home/$USER/go/src/github.com/metal3-io/baremetal-operator/config/manager/ | kubectl --kubeconfig=/tmp/kubeconfig-{{ CLUSTER_NAME }}.yaml apply -n {{ NAMESPACE }} -f -"
```

Comment:
I am thinking whether we should change the namespace here since it is now `baremetal-operator-system`

Reply:
@maelk ^